src/compiler/utilities.ts

```diff
@@ -4408,7 +4408,7 @@ namespace ts {
     }
 
     export function getJSDocTypeParameterDeclarations(node: DeclarationWithTypeParameters): readonly TypeParameterDeclaration[] {
-        return flatMap(getJSDocTags(node), tag => isNonTypeAliasTemplate(tag) ? tag.typeParameters : undefined);
+        return flatMap(getJSDocTags(node), tag => isNonTypeAliasTemplate(tag) ? tag.typeParameters.filter(p => !containsParseError(p)) : undefined);

```

Comment:
I heard u like Haskell
```suggestion
        return flatMap(getJSDocTags(node), tag => isNonTypeAliasTemplate(tag) ? tag.typeParameters.filter(not(containsParseError)) : undefined);
```

Reply:
Um....oops. This isn't needed since I found that the Pursuit fellows made a better fix in December.